packages/vapor/scss/redesign/variables.scss

```diff
@@ -111,6 +111,7 @@
 
     // Icon
     --documentation-link-icon-color: var(--digital-blue-60);
+    --tooltip-icon-color: var(--digital-blue-60);
```

Comment:
Currently, we have lots of CSS variables in several files and it gets confusing sometimes. The main reason we created CSS variables was to keep the old style and override with light reskin style when needed. Since now the new style is applied everywhere, I think we could try to use fewer CSS variables even remove existing CSS variables which are no longer reusable (no need to override). The only use case at this moment, AFAIK, is to override style by different `.mod-` in specific components, ex: in `badge.scss`